src/pages/People/People.tsx

```diff
@@ -0,0 +1,212 @@
+import react from 'react';
+import { useState, useEffect } from 'react'
+import './People.css'
+import img1 from '../../components/ProjectGallery/media/blank-profile-picture.webp'
+import axios from 'axios'
+import dotenv from 'dotenv';
+import { AiFillLinkedin, AiFillPhone } from "react-icons/ai";
+import { MdEmail } from "react-icons/md";
+
+dotenv.config();
+const baseURL = process.env.REACT_APP_API_URL;
+
+const People = () => {
+
+    interface MemberOBJ {
+        name: NameInfo,
+        project: string,
+        position: string,
+        contact: ContactInfo,
+        message? : string
+    }
+
+    interface ContactInfo {
+        email?: string,
+        linkedIn?: string,
+        phoneNumber?: string
+    }
+
+    interface NameInfo {
+        firstname: string,
+        lastname: string,
+    }
+
+    const dummyList: Array<string> = ['Supervisors', 'Correlation', 'IDEO', 'IT', 'NOVA', 'Perceptual Modes']
+
+    // I am not sure if this something that should be kept but this an example of what the array would look like from the backend
+    const memberDummyList: Array<MemberOBJ> = [
+        {
+            name: { firstname: 'John', lastname: 'Doe' },
+            project: 'Correlation',
+            position: 'Something1',
+            contact: { linkedIn: 'ryanyae', email: '123456@gmail.com', phoneNumber: "(123)-456-7890" }
+        },
+        {
+            name: { firstname: 'Jane', lastname: 'Doe' },
+            project: 'IDEO',
+            position: 'Something2',
+            contact: { linkedIn: 'ryanyae', email: '123456@gmail.com' }
+        },
+        {
+            name: { firstname: 'JKoe', lastname: 'Doe' },
+            project: 'IT',
+            position: 'Something3',
+            contact: { linkedIn: 'ryanyae', email: '123456@gmail.com' }
+        },
+        {
+            name: { firstname: 'Jane', lastname: 'Doe' },
+            project: 'NOVA',
+            position: 'Something4',
+            contact: { linkedIn: 'ryanyae', email: '123456@gmail.com' }
+        },
+        {
+            name: { firstname: 'Jane', lastname: 'Doe' },
+            project: 'Perceptual Modes',
+            position: 'Something5',
+            contact: { linkedIn: 'ryanyae', email: '123456@gmail.com' }
+        },
+        {
+            name: { firstname: 'Jane', lastname: 'Doe' },
+            project: 'Perceptual Modes',
+            position: 'Supervisors',
+            contact: { linkedIn: 'ryanyae', email: '123456@gmail.com' }
+        }
+    ]
+
+
+
+    // the page will be defaulted to bein on the first grid item
+    const [currentProject, setCProject] = useState<string>(dummyList[0])
```

Comment:
I think changing the name to `setCurrProject` would be amazing!

```suggestion
    const [currentProject, setCurrProject] = useState<string>(dummyList[0])
```